### DISCUSSION TITLE
how do you deal with 1 socket io polling [Sep 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3821] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Bergi9**  
#### nocodb has socket.io? and what do you mean with asset loading? i have nocodb running in docker and traefik as reverse proxy  
#  
> 🙋‍♂️ **emdax**  
#### if you upload attachments and reload it will pull absolute paths for me  
#  
> 🙋‍♂️ **emdax**  
#### if you make changes across two browsers i get socket.io errors for SSL  
#  
> 🙋‍♂️ **emdax**  
#### HTTPS with NGINX rewrite 80:8080 with Lets Encrypt SSL is my set up  
#  
> 🙋‍♂️ **Bergi9**  
#### https://docs.nocodb.com/getting-started/installation#environment-variables   
#### NC_PUBLIC_URL is used for generating absolute url for uploaded files  
#  
> 🙋‍♂️ **Bergi9**  
#### for nginx you need to specify websocket forward too  
#  
> 🙋‍♂️ **emdax**  
#### are there PROD deploy docs for this?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1019138920975450112)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
